ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -2797,6 +2801,574 @@ private CadTemplate readLeader()
+			int leaderLineCount = _objectReader.ReadBitLong();
+			for (int ll = 0; ll < leaderLineCount; ll++) {
+				leaderRoot.Lines.Add(readLeaderLine(template));
+			}
+
+			//R2010
+			//	BS	271	Attachment direction(0 = horizontal, 1 = vertical, default is 0)
+			leaderRoot.AttachmentDirection = (TextAttachmentDirectionType)_objectReader.ReadBitShort();
+			return leaderRoot;
+		}
+
+		private LeaderLine readLeaderLine(CadMLeaderTemplate template) {
+			LeaderLine leaderLine = new LeaderLine();
+			CadMLeaderTemplate.LeaderLineSubTemplate leaderLineSubTemplate = new CadMLeaderTemplate.LeaderLineSubTemplate(leaderLine);
+			template.LeaderLineSubTemplates.Add(leaderLineSubTemplate);
+
+			//	Points
+			//	BL	-	Number of points
+			//	3BD		10		Point
+			int pointCount = _objectReader.ReadBitLong();
+			for (int p = 0; p < pointCount; p++) {
+				leaderLine.Points.Add(_objectReader.Read3BitDouble());
+			}
+
+			//	Add optional Break Info (one or more)
+			//	BL	Break info count
+			leaderLine.BreakInfoCount = _objectReader.ReadBitLong();
+			if (leaderLine.BreakInfoCount > 0) {
+				//	BL	90		Segment index
+				leaderLine.SegmentIndex = _objectReader.ReadBitLong();
+
+				//	Start/end point pairs
+				//	3BD	11	Start Point
+				//	3BD	12	End point
+				int startEndPointCount = _objectReader.ReadBitLong();
+				for (int sep = 0; sep < startEndPointCount; sep++) {
+					leaderLine.StartEndPoints.Add(new StartEndPointPair(
+						_objectReader.Read3BitDouble(),
+						_objectReader.Read3BitDouble()));
+				}
+			}
+
+			//	BL	91	Leader line index
+			leaderLine.Index = _objectReader.ReadBitLong();
+
+			//	R2010
+			//	BS	170	Leader type(0 = invisible leader, 1 = straight leader, 2 = spline leader)
+			leaderLine.PathType = (MultiLeaderPathType)_objectReader.ReadBitShort();
+			//	CMC	92	Line color
+			leaderLine.LineColor = _objectReader.ReadCmColor();
+			//	H	340	Line type handle(hard pointer)
+			leaderLineSubTemplate.LineTypeHandle = this.handleReference();
+			//	BL	171	Line weight
+			leaderLine.LineWeight = (LineweightType)_objectReader.ReadBitLong();
+			//	BD	40	Arrow size
+			leaderLine.ArrowSize = _objectReader.ReadBitDouble();
+			//	H	341	Arrow symbol handle(hard pointer)
+			leaderLineSubTemplate.ArrowSymbolHandle = this.handleReference();
+
+			//	BL	93	Override flags (1 = leader type, 2 = line color, 4 = line type, 8 = line weight, 16 = arrow size, 32 = arrow symbol(handle)
+			leaderLine.OverrideFlags = (LeaderLinePropertOverrideFlags)_objectReader.ReadBitLong();
+
+			return leaderLine;
+		}
+
+
+		private void showCurrentPosAndShift() {
+			long positionO = _objectReader.Position;
+			int bitShiftO = _objectReader.BitShift;
+			System.Diagnostics.Debug.WriteLine($"Pos: {positionO}, bShi: {bitShiftO}");
+			long positionH = _handlesReader.Position;
+			int bitShiftH = _handlesReader.BitShift;
+			System.Diagnostics.Debug.WriteLine($"Pos: {positionH}, bShi: {bitShiftH}");
+		}
+
+
+		private void analyse02(int count = 20) {
```

Comment:
If I'm not mistaken the methods `analyse02` and `analyse03` are implemented to explore the stream values, please do not include them in the final PR.